examples/03-tips-n-tricks/03-using_inline_functions_and_Query.py

```diff
@@ -0,0 +1,116 @@
+"""
+.. _ref_how_to_use_Query:
+
+Using inline functions (Query)
+-----------------------------
+
+Inline functions like ``UX`` have been implemented in PyMAPDL as methods
+on the ``mapdl.inline_functions.Query`` object. In this example we set
+up a simple simulation and use ``Query`` to demonstrate some of its
+functionality.
+
+We create a ``Query`` instance as below, using the ``mapdl`` instance
```

Comment:
```suggestion
We create an instance of :class:`ansys.mapdl.core.inline_functions.Query` below, using
the ``mapdl`` instance.
```
This will provide links to existing class documentation.  If we're missing this or the link doesn't work, we need to add this to a subsequent PR.